src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -157,7 +157,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_DEPLOYMENTSCRIPTS: '2020-10-01',
         ResourceType.MGMT_RESOURCE_TEMPLATESPECS: '2019-06-01-preview',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
-        ResourceType.MGMT_KEYVAULT: '2020-04-01-preview',
+        ResourceType.MGMT_KEYVAULT: '2021-04-01-preview',
```

Comment:
It is not reasonable to mark `az keyvault` as GA while the underlying API is preview, not to mention `2021-04-01-preview` doesn't even have REST API doc:

https://docs.microsoft.com/en-us/rest/api/keyvault/vaults/get

![image](https://user-images.githubusercontent.com/4003950/115524487-c657de80-a2c0-11eb-983c-a902cace445e.png)

Reply:
This is more like a problem across all CLI command modules. I don't have a good idea about this